writing-app/src/App.js

```diff
@@ -1,16 +1,56 @@
 import React, { Component } from 'react'
 import logo from './logo.png'
 import './App.css'
-import Prompt from './components/Prompt'
+// import Prompt from './components/Prompt'
+import {PROMPTS} from './shared/prompts.js'
 
+
+
+class Prompt extends Component {
```

Comment:
There's the Prompt class defined in the `src/components/Prompt.js` file. Is there a reason you defined it here again?

Reply:
That file should be deleted. I moved the Prompt component directly inside the App.js for convenience.